app/frontend/src/components/forms/submission/MySubmissionsTable.vue

```diff
@@ -2,11 +2,12 @@
   <div>
     <v-skeleton-loader :loading="loading" type="heading">
       <v-row class="mt-6" no-gutters>
-        <v-col class="text-center" cols="12" sm="10" offset-sm="1">
+        <!-- page title -->
+        <v-col cols="12" sm="6" order="2" order-sm="1">
           <h1>Previous Submissions</h1>
-          <h2>{{ formId ? form.name : 'All Forms' }}</h2>
         </v-col>
-        <v-col class="text-right" cols="12" sm="1">
+        <!-- buttons -->
+        <v-col class="text-right text-sm-right" cols="12" sm="6" order="1" order-sm="2">
```

Comment:
Do we need `text-sm-right`? IIRC `text-right` implicitly means `text-xs-right` which would also include `text-sm-right`. Can't remember if that's the case or not.

Reply:
Correct, will be changing this in the next commit. Also going to be doing this change for all the other places we are using this.